src/index.js

```diff
@@ -210,7 +265,40 @@ class App extends React.Component {
       );
     };
 
-    return (
+    return <>
+      <div className="exportWrapper">
```

Comment:
Not for this PR but we’ll likely want to hide all those options by default and only show them when the person actually clicks export

Reply:
Yep. I'm planning to improve the export in the future (custom filename, ...).